spec/policies/aggregation_policy_spec.rb

```diff
@@ -87,7 +87,7 @@
           expect(resolved_scope).to include(public_aggregation)
         end
 
-        it 'includes aggregations from owned private projects' do
+        xit 'includes aggregations from owned private projects' do
```

Comment:
what are the plans with skipped tests? Mainly asking because I can see this easily being forgotten in history as skipped and never addressed again. 

If the plans are to delete these tests, maybe we can do that now? Or maybe the batch aggregation PR will take care of these skipped tests removals/give these specs more context so that we can un-skip the tests. 

Maybe adding a comment or something before the skipped tests to explain what the plan is would be helpful. BUT if this is quickly mitigated by the followup PR, them we can leave as is.

Reply:
Everything related to aggregations is being refactored by the batch aggregation PR. These bits also had to be skipped here because the notion of a private project was handled by the attribute that this removes. I'll remove them if you prefer, either way it's all getting removed when that PR merges (hopefully very soon!).